test/downloadutils.js

```diff
@@ -25,6 +25,16 @@ var https = require('https');
 function downloadFile(file, url, callback, redirects) {
   var completed = false;
   var protocol = /^https:\/\//.test(url) ? https : http;
+
+  if (url.indexOf('web.archive.org') !== -1) {
+    // Web Archive URLs need to be transformed to add `if_` after the ID.
+    // Without this, an HTML page containing an iframe with the PDF file
+    // will be served instead (issue 8920).
+    var regex = /web\.archive\.org\/web\/(\d+)/g;
```

Comment:
minor optimization, make `regex` global outside `downloadFile` to prevent to compile it each time ?